cluster_agent/identity/slurmrestd.py

```diff
@@ -31,15 +31,11 @@ def _load_token_from_cache(username: str) -> typing.Union[str, None]:
         token = token_path.read_text().strip()
         logger.debug(f"Retrieved token from {token_path} as {token}")
     except Exception:
-        logger.warning(
-            f"Couldn't load token from cache file {token_path}. Will acquire a new one"
-        )
+        logger.warning(f"Couldn't load token from cache file {token_path}. Will acquire a new one")
         return None
 
     try:
-        jwt.decode(
-            token, options=dict(verify_signature=False, verify_exp=True), leeway=-10
-        )
+        jwt.decode(token, options=dict(verify_signature=False, verify_exp=True), leeway=-10)
```

Comment:
Can we use jose to decode the tokens as well? It should be very straightforward and would eliminate some redundancy.

Reply:
actually, I will apply this change in this PR